### ISSUE TITLE
[BUG] RO-Crate export missing MUST fields

### ISSUE THREAD

**Issue by floWetzels** [nfdi4plants/ARCtrl#472]
The investigation object in the RO-Crate export does not necessarily have a `license` and `name` field, however, they are required by the RO-Crate specification.

Could the gitlab CI detect an existing license and insert it into the arc export command? @j-bauer

**Comment by feserm** [comment]
There are some available APIs to query against:
- https://github.com/licensee/licensee
- https://docs.github.com/de/rest/licenses?apiVersion=2022-11-28

When searching for license scanning and gitlab this popped up:
- https://github.com/licensee/licensee

**Comment by j-bauer** [comment]
Yes, this can be done in the CI-Pipeline. Can you point me to an example RO-Crate containing a license?

**Comment by HLWeil** [comment]
I think we don't have any yet. But as I said I could insert this as an argument in the arc-export tool, so you would only need to provide this when calling the export

**Comment by HLWeil** [comment]
`name` part done but `license` missing? @floWetzels

**Comment by floWetzels** [comment]
Correct, PR https://github.com/nfdi4plants/ARCtrl/pull/488 changed `headline` to `name` in Investigations. Licenses are still not handled at all, as far as I know.

I guess we should also change `headline` to `name` for studies for consistency. Also, studies can be seen as their own datasets, so I guess it makes sense for them to have a name on their own. There is no field in assays though that can be mapped to `name`. What do you think, @HLWeil?

**Comment by HLWeil** [comment]
Yup makes sense 👍

**Comment by kMutagene** [comment]
> Correct, PR https://github.com/nfdi4plants/ARCtrl/pull/488 changed headline to name

IOn the future, please first update the profile, i am currently aggregating needed changes on the profile and specs, and the codebase being ahead of the profile is something i did not expect.

**Comment by floWetzels** [comment]
> > Correct, PR [#488](https://github.com/nfdi4plants/ARCtrl/pull/488) changed headline to name
> 
> In the future, please first update the profile, i am currently aggregating needed changes on the profile and specs, and the codebase being ahead of the profile is something i did not expect.

That's not the case though, see https://github.com/nfdi4plants/isa-ro-crate-profile/pull/36

**Comment by kMutagene** [comment]
Yeah i Messe up looking at the Release branch sorry

________________________________

**Comment by HLWeil** [comment]
Logic for inclusion and default values included since #496.

Will open up an Issue in https://github.com/nfdi4plants/arc-export for allowing setting a license value.
